calculator.js

```diff
@@ -1,28 +1,61 @@
 
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
-}
-// '3 + 5 = 8'
+const calculateAdd = function(input) {
+  let result = input.num1 + input.num2
+  console.log(`${input.num1} + ${input.num2} = ${result}`);
+   return;
```

Comment:
In addition, since result is not reassigned in this scope, it is best to use `const`

```suggestion
  const result = input.num1 + input.num2
  console.log(`${input.num1} + ${input.num2} = ${result}`);
  return result;
```